### ISSUE TITLE
update Urho3D-CMake-common.cmake

### ISSUE THREAD

**Issue by bvanevery** [gawag/Urho-Sample-Platformer#13]
Urho3D's version of Urho3D-CMake-common.cmake is quite different than the one used by Urho3D Sample Platformer.  I suspect it leads to build errors.  Particularly, one I'm currently trying to chase down pertains to resource path prefix handling, and copying resources to the CMake build directory when USP is built "out of source".  Even if that doesn't turn out to be the solution to the problem, I bet Urho3D has probably fixed various bugs and changed how things are supposed to behave.

**Comment by bvanevery** [comment]
Not wanting to understand things so much as get them to work, I opted to replace the entire \CMake directory with the stuff from the most recent Urho3D.  Also deleted the top level FindUrho3D.cmake file as it duplicates the one contained in \CMake\Modules.  Also moved Build\bin to \bin so that resources would be correctly copied for an out-of-source CMake build.  With these changes and others filed in other issues, I can now run USP.  However all I've got is a menu that gives big button choices of "Play" and "Quit".  I'm not entirely sure what USP is supposed to be capable of, so I'm guessing there is something still wrong with acquiring resources.
